tests/rt.conf

```diff
@@ -77,7 +77,6 @@ RUN     | control_thompson
 RUN     | control_thompson_no_aero                                                                                                | - wcoss_cray                            | fv3 |
 
 RUN     | fv3_HAFS_v0_hwrf_thompson                                                                                               |                                         | fv3 |
-#RUN     | fv3_HAFS_v0_hwrf                                                                                                       |                                         | fv3 |
```

Comment:
See the change in line 66 - we don't compile the suite anymore, and this PR removes it. We should note, however, that this is the last suite that contains Ferrier-Aligo MP, which means that it is no longer tested to compile (we didn't test it to run correctly for a long time, because it doesn't). That may be ok, but we should be aware of it.